### ISSUE TITLE
Status Bar's append text option doesn't work

### ISSUE THREAD

**Issue by CyberBotX** [xiaoxiaoflood/firefox-scripts#206]
I am not sure when this started to happen, but for a while now, if I change the status bar to append text (that is, setting `userChromeJS.statusbar.appendStatusText` in `about:config` to `true`, which I know is the default), I do not see any text in the status bar. The only machine of mine where this has been working is my FreeBSD machine, but for every single Windows install of Firefox, it has not been working. I have experienced the problem even when adding the script to a brand new profile in Firefox on every Windows install I've done this on. I think some older versions of the status bar might work, but I don't have the time to check all the older versions.

**Comment by xiaoxiaoflood** [comment]
I can't reproduce. It's fine for me both on my daily profile and also on a fresh new profile I just created to test.

**Comment by xiaoxiaoflood** [comment]
Closing as unconfirmed. If you find a way to get steps to reproduce from a new profile please comment and I'll see.

**Comment by CyberBotX** [comment]
Sorry I never got back to this, but I just decided to see what it would take to reproduce this from a new profile. At first, I just copied the required files into my chrome directory and the status bar was working fine. I also copied over some custom CSS that uses userChrome.css and that didn't break it. I started to install extensions I was using and it didn't break... until I installed Firefox Color. Once I had Firefox Color installed, the status bar stopped displaying the text. If I disable Firefox color without uninstalling it, the status bar text comes back. I don't know enough about the status bar JavaScript or the Firefox Color extension, but something is not playing nice between the two of them.

**Comment by CyberBotX** [comment]
Once again apologies for not getting to this again. I think I've determined the following:

The status bar text IS working, but for whatever reason, it is outputting as white on my Windows machine but black on my FreeBSD machine. Because it outputs as white on Windows, when using a light theme (which includes the theme I was using with Firefox Color as well as the built-in Light 1.2 theme), it looks as if there is no text when it actually is there.

From inspecting the code of the status bar as well as using the Browser Toolbox, I found that for
whatever reason, my Firefox is overriding the color set in the code away from `#444` to `rgb(249, 249, 250)`. I found the fix was to change the color on the `#status-text > #statuspanel-label` selector to `var(--toolbar-non-lwt-textcolor) !important`.
